### PR TITLE
description update

### PR DESCRIPTION
**Pull request by Ni-Knight**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11909

## Description
Updated the description and tooltips with clearer data

### CODE REVIEW

**Review by efelmandar — APPROVED** [review]

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 1]
```diff
@@ -61,3 +61,45 @@ The integration allows the use of basic authentication in the requests.
 To enable basic authentication, a user and password have to be supplied in the Credentials parameters in the integration configuration.
 
 The server will then authenticate the requests by the `Authorization` header, expecting basic authentication encrypted in base64 to match the given credentials.
+
+
+## STIX types for STIX indicator Domain Object
+
+Some tools require the indicators in STIX to be a STIX indicator type and not an SCO. If that is the case you can select which indicator types will be converted into SDOs using a STIX Pattern for the indicator value.
+
+For example, when `STIX types for STIX indicator Domain Object` is not selected for IPs XSOAR will output the IP indicators as an SCO:
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:70]:
```suggestion
For example, when `STIX types for STIX indicator Domain Object` is not selected for IPs, Cortex XSOAR will output the IP indicators as an SCO:
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 2]
```diff
@@ -61,3 +61,45 @@ The integration allows the use of basic authentication in the requests.
...
+            "modified": "2024-10-01T07:07:00.440958Z",
+            "spec_version": "2.1",
+            "type": "ipv4-addr",
+            "value": "192.168.1.1"
+        }
+    ]
+}
+```
+
+When `ipv4-addr` is selected in `STIX types for STIX indicator Domain Object` XSOAR will output the indicator as an Indicator SDO with the correct pattern:
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:86]:
```suggestion
When `ipv4-addr` is selected in `STIX types for STIX indicator Domain Object`, Cortex XSOAR will output the indicator as an Indicator SDO with the correct pattern:
```

**Comment by ShirleyDenkberg** [comment]
@efelmandar Doc review completed.

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md (hunk 1 above)
Comment (Ni-Knight) [reply Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md]: I've removed the word XSOAR completely as XSIAM is also affected.

**Comment by Ni-Knight** [comment]
Force Merge - This is a description and tooltip update only

**Pull request merged**